variables.tf

```diff
@@ -101,6 +101,12 @@ variable "private_s3_endpoint" {
   description = "Deploy an S3 endpoint for your private subnets"
 }
 
+variable "restrict_default_security_group" {
+  type        = bool
+  default     = true
+  description = "Remove all rules from the default security group"
+}
```

Comment:
What will be the impact of this change for already running environments using this module?

Reply:
Impact will be it tries to remove all rules from the default security group per default. 

I can default the value to false indeed and 0 impact, but then you have to make the choice for setting variable if you want to be CIS compliant. 

I'm not sure if 'being CIS compliant per default' is what we are striving for with these modules, so let me know if I should default it false. I guess the other option is 'major version change' for a minor code change?

Reply:
ive defaulted this to false now in the interest of moving this forward, I guess default discussions can be had seperately.

Reply:
I would default this to `true` as I believe majority are creating their own security groups to manage with Terraform anyway.